modules/installing-ocp-agent-inputs.adoc

```diff
@@ -170,24 +169,3 @@ You must provide the rendezvous IP address when you do not specify at least one
 <3> Optional: Overrides the hostname obtained from either the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup. Each host must have a unique hostname supplied by one of these methods.
 <4> Enables provisioning of the Red Hat Enterprise Linux CoreOS (RHCOS) image to a particular device. It examines the devices in the order it discovers them, and compares the discovered values with the hint values. It uses the first discovered device that matches the hint value.
 <5> Optional: Configures the network interface of a host in NMState format.
-
-ifdef::pxe-boot[]
-
-. Optional: To create an iPXE script, add the `bootArtifactsBaseURL` to the `agent-config.yaml` file:
-+
-[source,yaml]
-----
-apiVersion: v1beta1
-kind: AgentConfig
-metadata:
-  name: sno-cluster
-rendezvousIP: 192.168.111.80
-bootArtifactsBaseURL: <asset_server_URL>
-----
-+
-Where `<asset_server_URL>` is the URL of the server you will upload the PXE assets to.
-endif::pxe-boot[]
-
-ifeval::["{context}" == "prepare-pxe-assets-agent"]
-:!pxe-boot:
-endif::[]
```

Comment:
This is content that I merged earlier, which triggered the conflict, but this should be included in the main branch of the docs. Can you add this content back in, and then squash all of your commits into one?